src/libxtp/gwbse/gw.cc

```diff
@@ -168,19 +174,56 @@ void GW::CalculateGWPerturbation() {
         << TimeStamp() << " Calculated screening via RPA" << std::flush;
     XTP_LOG(Log::info, _log)
         << TimeStamp() << " Solving QP equations " << std::flush;
+    if (_opt.gw_mixing_order > 0 && i_gw == 1) {
```

Comment:
why equals 1?

Reply:
Because I don't think it makes sense to have the dft energies in the history for mixing.

Reply:
why then >0 instead of ==1?

Reply:
That does work with another change.